echopype/echodata/simrad.py

```diff
@@ -0,0 +1,233 @@
+"""
+Contains functions that are specific to Simrad echo sounders
+"""
+from typing import Optional, Tuple
+
+from .echodata import EchoData
+
+
+def _check_mode_input_without_data(
```

Comment:
How about `_check_input_args_combination`?

Reply:
I am fine with this name. I will go ahead and change it.